### PR TITLE
Added note about address validation on send

### PR DESCRIPTION
**Pull request by thrawn01**

### CODE REVIEW

File: source/user_manual.rst [hunk 1]
```diff
@@ -192,7 +192,11 @@ When sending via HTTP API, Mailgun offers two options:
 - You can submit the individual parts of your messages to Mailgun, such as text
   and html parts, attachments, and so on. This doesn't require any MIME knowledge on your part.
 
-.. note:: Mailgun supports maximum messages size of 25MB.
+.. note:: - Mailgun supports maximum messages size of 25MB.
+          - HTTP and SMTP sends preform rudimentary email address verification by preforming
+            syntax checks according to RFC5321, RFC5322, RFC6854, and preforming a DNS verification
+            of the recipient domain.
```
Comment (censhin) [inline source/user_manual.rst:198]: Maybe reword this a little as you use `perform` three times. Also, it's misspelled on line 197.

File: source/user_manual.rst [hunk 2]
```diff
@@ -192,7 +192,11 @@ When sending via HTTP API, Mailgun offers two options:
 - You can submit the individual parts of your messages to Mailgun, such as text
   and html parts, attachments, and so on. This doesn't require any MIME knowledge on your part.
 
-.. note:: Mailgun supports maximum messages size of 25MB.
+.. note:: - Mailgun supports maximum messages size of 25MB.
+          - HTTP and SMTP sends preform rudimentary email address verification by preforming
```
Comment (highlycaffeinated) [inline source/user_manual.rst:196]: s/preforming/performing/

**Review by censhin — CHANGES_REQUESTED** [review]

File: source/user_manual.rst [hunk 3]
```diff
@@ -192,7 +192,11 @@ When sending via HTTP API, Mailgun offers two options:
...
+            syntax checks according to RFC5321, RFC5322, RFC6854, and preforming a DNS verification
```
Comment (highlycaffeinated) [inline source/user_manual.rst:197]: s/preforming/performing/

**Review by highlycaffeinated — CHANGES_REQUESTED** [review]

**Review by censhin — APPROVED** [review]
LGTM
